index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
+  <title>Trivia Game</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="center-align">
+    <h2 class="gameName">Animal Facts Trivia Game!</h2>
+    <button class="btn waves-effect waves-light" type="submit" name="action" id="startGame">Start Game</button>
+  </div>
+  <div class="container center-align">
+    <div class="row">
+      <h4>Time Left: <span class="timer"></span></h4>
+      <h4>Question: <span class="questionTitle"></span></h4>
+      <h4 class="message"></h4>
+      <h4 class="answerTitle">The correct answer is: </h4>
+      <div class="answers">
+        <div class="card-action">
+          <p>
+            <label>
+              <input name="group1" class="choices" type="radio"/>
```

Comment:
you could also display these dynamically instead of hard coding in the choices. The same way you created the questions 👍